doc/builders/images.xml

```diff
@@ -10,4 +10,5 @@
  <xi:include href="images/ocitools.section.xml" />
  <xi:include href="images/snaptools.section.xml" />
  <xi:include href="images/portableservice.section.xml" />
+ <xi:include href="images/makediskimage.section.xml" />
```

Comment:
Did you change anything about the interface? Otherwise I would pull this out into its own PR.

Reply:
Yes, I added parameters.